src/Integration/MefServices/VsSessionHost.cs

```diff
@@ -58,16 +56,14 @@ internal sealed class VsSessionHost : IHost, IProgressStepRunnerWrapper, IDispos
         [ImportingConstructor]
         public VsSessionHost([Import(typeof(SVsServiceProvider))] IServiceProvider serviceProvider,
             ISonarQubeService sonarQubeService,
-            IActiveSolutionTracker solutionTacker,
-            IProjectToLanguageMapper projectToLanguageMapper,
```

Comment:
The only reason the host needed the `projectToLanguageMapper` was to create its own instance of `ProjectSystemHelper` -> no longer required.

The host doesn't now reference `IProjectSystemHelper` at all.